spec/human_spec.rb

```diff
@@ -9,6 +9,16 @@
   let(:ask_for_move) { 'Please enter a number between 1 - 9: ' }
   let(:spot_taken) { ['-', '-', 'X'] }
 
+  describe 'exit?' do
+    it 'returns false when user does not input "exit"' do
+      expect(human.exit?('1')).to eql(false)
+    end
+
+    it 'returns true when user inputs "exit"' do
+      expect(human.exit?('exit')).to eql(true)
+    end
```

Comment:
Is the user aware that they can input `exit` to quit the game?

Reply:
Added this (along with some game play instructions) to the welcome message.

Reply:
Hoping this will eventually be in a script class (which is requested on another branch).